SkyBlock/README.md

```diff
@@ -0,0 +1,358 @@
+# SkyBlock
+
+![](Skyblock_Island.webp "SkyBlock Island")
+
+## Introduction
+
+**Skyblock** is one of the most popular survival gamemodes. The idea is that you can create and expand your very own world infinitely with only minimal materials. You start on a small island floating in the middle of void with a few items and a single tree and you must learn to use and preserve your materials wisely as each one has a strategic use, (for example lava buckets), and to know the basics of Minecraft in order to complete challenges and to expand. It is normal to make beginner mistakes, sometimes requiring many restarts.
+
+## Commands
+
+On TrixMC we have a set of commands that you can use to manage your SkyBlock world. The commands are of the format `/is <command>`. The commands are listed below:
+
+### controlpanel
```

Comment:
```suggestion
### `controlpanel`
```
make all the titles also codeblocks, looks nicer imo

Reply:
Done